### DISCUSSION TITLE
Week 12 Project Updates

### DISCUSSION THREAD

**Discussion by CJ3315** [USU-ITLS-x265-23-summer/class-repo#220] [General]
This week I mainly focused on my contact page with the forms and information. 
**Tasks Completed**

- Added Contact info:
- Phone, email, company, location 
- Created a contact form for viewers to fill out, info included: 
- name 
- email 
- Subject 
- message 

**To do:** 
Get feedback and final touches!
**I've added some fairly zoomed-out screenshots, I would love some feedback on the design elements and layout- Thank you!**
Link to the project page: https://github.com/CJ3315/Portofolio-Project
![image](https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/133183018/ac9546de-49a2-405e-ae6f-77ad2ef9505e)
![image](https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/133183018/4bdf520c-27bf-45b6-90e9-3e4a80c522a2)
![image](https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/133183018/aebd8570-4221-4c7f-9604-3ce7e0e1ab18)
![image](https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/133183018/69771d41-2382-400b-ad1a-99301fdf1774)
![image](https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/133183018/212a3f80-836b-416f-bb4b-64fec120f9ae)

**Comment by kassyoveson** [comment]
This looks amazing CJ. Great Job. Some suggestions I have for you is to move the "I am a strong supportive leader" box further to the right of the page. Not sure if its the screen, but the New Years box looks like it is not the correct size. I also suggest to use better contrasts for the contact box information. I would suggest not using black, but a different color.

**Comment by gabebott** [comment]
This looks awesome! I love the colors that you chose to include in your site, and I think that it shows a lot of personality and character in your design. I love the thumbnail inclusions, I think you could add titles to the videos which would make it even better. Great job!

**Comment by derekscholes** [comment]
This does look good, I agree with previous suggestions as well. Another thing that would be nice is if you can publish the site with GitHub. that way we can interact with it as well and see your progress. One important thing I have found is that I needed to look at my site on multiple devices and screen sizes to see how the page changes based off of the two. In some cases it looked horrible and nothing could be read.

**Reply by derekscholes** [reply]
I just used a method to look at your page and it is very good. I really like it. Very eye appealing and user friendly. The only issue I had was that some symbols didn't show up for me and I don't know if that is an error that is coming up from your repo or just the fact of how I was looking at your page.

**Comment by beto0890** [comment]
Your project looks fantastic! Great background colors and sections. The only thing that I would consider will be on the contact me page with a black background it is really hard to read the words inside each section and the font size could be bigger.
